src/arena.rs

```diff
@@ -575,3 +576,94 @@ where
         })
     }
 }
+
+/// A set of arena handles, represented as a bit vector.
+///
+/// This represents a set of `Handle`s simply by storing a bit for every
+/// `Handle` index. The underlying bit vector is sized dynamically, so storage
+/// depends on the highest handle index that has been inserted.
+///
+/// A `HandleBitSet<T>` will only hold `Handle<T>` values, so it's a bit more
```

Comment:
is the strict typing useful here, considering that it's only used in the validator and the backends? I.e. it's hardly a part of a public API